Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: client-sdks/advanced/query-json-in-sqlite.mdx

---
title: "Querying JSON Data in SQLite"

> description: "Query JSON data synced from your backend and stored as strings in SQLite using json_extract, JSON operators, and indexing strategies."

Reviewer: ```suggestion
description: "How to query JSON data synced from your backend and stored as strings in SQLite using json_extract and JSON operators."
```
